src/appInsights.spec.ts

```diff
@@ -18,6 +22,14 @@ describe('appInsights', () => {
   it('doesn\'t add -dev label to version logged in the telemetry when CLI installed from npm', () => {
     sinon.stub(fs, 'existsSync').callsFake(() => false);
     const i: any = require('./appInsights');
+    console.log(i.default.commonProperties.version);
```

Comment:
Seems like something we should remove :)

Reply:
My bad, now removed.